provider/storage_benchmark.py

```diff
@@ -213,6 +213,7 @@ def _find_exe_file():
         if not utils_misc.wait_for(
                 lambda: _find_exe_file(), timeout, step=3.0):
             raise TestError('Failed to install fio under %.2f.' % timeout)
+        self.session.close()
```

Comment:
From my checking, the `session` is useful in the following steps,  such as clean_env, kill_process and so on.
I am afraid it can not be closed directly.

Reply:
To avoid the problem that clean env failed as resource is busy (return value is true).  Suggest update the function `_find_exe_file`   remove `CD $dst` and use `DIR` directly.

Reply:
Ok, will update it, thank you

Reply:
@yanan-fu Updated the code, could you help to review it agian?

Thanks~